### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List - 
        app-checkin-location/csharp
        app-installation-using-qr-code/csharp
        bot-proactive-messaging/csharp
        bot-release-management/csharp
        graph-bulk-meetings/csharp**

### CODE REVIEW

**Review by Alive-Fish — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]
The env variable name is not critical issue, you can update in next PR.

File: samples/graph-bulk-meetings/csharp/README.md [hunk 1]
```diff
@@ -89,11 +103,11 @@ Modify the `appsettings.json` file with the following details:
     - Update configuration with the ```MicrosoftAppId```, ```MicrosoftAppPassword``` and ```MicrosoftAppTenantId``` values received while doing Microsoft Entra ID app registration in your Azure portal.
     - Press `F5` to run the project
 
-1) Modify the `manifest.json` in the `/AppManifest` folder 
+1) Modify the `manifest.json` in the `/appPackage` folder 
 Replace the following details:
     - `<<Your Microsoft App Id>>` with your MicrosoftAppId received after doing Microsoft Entra ID app registration in your Azure portal.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/graph-bulk-meetings/csharp/README.md]: I have updated the below as per your comments.
image

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Review by Jegadeesh-MSFT — APPROVED** [review]
Looks Good !

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/12179362190

**Pull request merged**